src/platform_impl/linux/mod.rs

```diff
@@ -613,7 +613,7 @@ impl<T: 'static> EventLoop<T> {
             "Failed to initialize any backend! Wayland status: {:?} X11 status: {:?}",
             wayland_err, x11_err,
         );
-        panic!(err_string);
+        std::panic::panic_any(err_string);
```

Comment:
Considering that panic uses the same syntax as `format!`, which is used above, I believe this could be and probably should be written as
```rust
        panic!(
            "Failed to initialize any backend! Wayland status: {:?} X11 status: {:?}",
            wayland_err, x11_err,
        );
```